WatchDogEclipsePlugin/WatchDogUnitTests/src/nl/tudelft/watchdog/logic/interval/JsonConverterTest.java

```diff
@@ -65,7 +65,7 @@ public void testJsonTypingIntervalMissingDocumentRepresentation() {
 		ArrayList<IntervalBase> intervals = createSampleIntervals(interval);
 
 		assertEquals(
-				"[{\"doc\":{\"pn\":\"f6f4da8d93e88a08220e03b7810451d3ba540a34\",\"fn\":\"90a8834de76326869f3e703cd61513081ad73d3c\",\"sloc\":1,\"dt\":\"pr\"},\"it\":\"ty\",\"ts\":1,\"te\":2,\"ss\":\"\","
+				"[{\"modCount\":0,\"startEndDiff\":0,\"doc\":{\"pn\":\"f6f4da8d93e88a08220e03b7810451d3ba540a34\",\"fn\":\"90a8834de76326869f3e703cd61513081ad73d3c\",\"sloc\":1,\"dt\":\"pr\"},\"it\":\"ty\",\"ts\":1,\"te\":2,\"ss\":\"\","
```

Comment:
Would it be possible to add a few testcases to showcase the newly added features? (At least the startEndDiff.)